app/controllers/feedbacks_controller.rb

```diff
@@ -9,9 +9,8 @@ def new
 
   def create
     @feedback = Feedback.new(feedback_params)
-    if visit = session[:visit]
-      @feedback.prison = visit.prisoner.prison_name
-    end
+    @feedback.prison = prison_name if visit
```

Comment:
Actually not, the `visit` comes from [[here](https://github.com/ministryofjustice/prison-visits/blob/master/app/controllers/application_controller.rb#L64)]